src/scripts/main.js

```diff
@@ -1,3 +1,81 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('.field');
+const rowButtons = document.querySelectorAll('[class*="row"]');
+const columnButtons = document.querySelectorAll('[class*="column"]');
+const rows = [...table.querySelectorAll('tr')];
+const column = rows[0].cells;
+let rowsNumber = rows.length;
+let columnNumber = column.length;
```

Comment:
The variable 'column' is not used anywhere else in the code. If it's not intended for future use, consider removing it to avoid confusion and keep the code clean.